simple_history/models.py

```diff
@@ -30,12 +30,14 @@ class HistoricalRecords(object):
     def __init__(self, verbose_name=None, bases=(models.Model,),
                  user_related_name='+', table_name=None, inherit=False,
                  history_id_field=None,
+                 history_change_reason_field=None,
                  excluded_fields=None):
```

Comment:
I don't think anyone will use all of these args positionally but if someone is, existing callers of `excluded_fields` will break.

Reply:
Good point. Didn't catch this in #368 but we should probably reorder it so that `excluded_fields` comes before `history_id_field` and `history_change_reason_field`

Reply:
@krisneuharth see @kseever's comment here about positional args. Should probably add this after the other positional args.